model/registry/deprecated/process.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: registry.process.deprecated
+    type: attribute_group
+    brief: "Deprecated process attributes."
+    attributes:
+      - id: process.cpu.state
+        brief: >
+          The CPU state of the process.
```

Comment:
nit: 
```suggestion
          Deprecated, use `cpu.state` instead.
```

Reply:
Usually I wait for the person raised them to resolve them :) but yes those should be fixed.